.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.0
```

Comment:
Gemsets are just for development. It just keeps the gems installed isolated from other projects so you don't get conflicting dependencies (this was a problem for me when working on multiple rails projects). I've done it a few different way with `.ruby-version` to specify the version and then just use a `.ruby-gemset` to specify the gemset. However you can combine them into one file which is what mine is currently. I can switch to multiple files. :ship: as is